ui/auth/credentials.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package auth
+
+import (
+	"github.com/asdine/storm"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// Storage for Credentials
+type Storage interface {
+	StoreOrUpdate(bucket string, data interface{}) error
+	GetOneByField(bucket string, fieldName string, key interface{}, to interface{}) error
+}
+
+const (
+	initialUsername          = "myst"
+	initialPassword          = "mystberry"
+	credentialsStorageBucket = "app-credentials"
+	credentialsStorageID     = 1
+)
+
+// Credentials verifies/sets user credentials for web UI
+type Credentials struct {
+	username, password string
+	db                 Storage
+}
+
+// storeCredentials - struct stored in DB
+type storeCredentials struct {
+	ID           int
```

Comment:
Do we really need an `ID`, maybe it's enough to use a username as a key?

Reply:
As we have only one set of credentials, I found it the easiest to always store/update record with `ID=1`. Do you have other thoughts?

Reply:
Hm, for me keeping it as `username:passwordHash` looks simpler, you don't need extra `storeCredentials` struct, and username is a const, so we have only a single user always.

Reply:
Changed to key value store as discussed